web/server/entrypoint_arm64.go

```diff
@@ -5,18 +5,23 @@ package server
 import (
 	"go.viam.com/rdk/gostream"
 	"go.viam.com/rdk/gostream/codec/h264"
-	"go.viam.com/rdk/gostream/codec/h264/ffmpeg/avcodec"
 	"go.viam.com/rdk/gostream/codec/opus"
 	"go.viam.com/rdk/gostream/codec/x264"
 )
 
 func makeStreamConfig() gostream.StreamConfig {
 	var streamConfig gostream.StreamConfig
-	if avcodec.FindEncoderByName(h264.V4l2m2m) != nil {
-		streamConfig.VideoEncoderFactory = h264.NewEncoderFactory()
-	} else {
+
+	// Attempt to create a new encoder with hardcoded parameters
+	// to check if V4l2m2m codec is supported.
+	width, height, keyFrameInterval := 1920, 1080, 30
+	_, err := h264.NewEncoder(width, height, keyFrameInterval, nil)
```

Comment:
Not passing in logger as it is preferable to suppress error logs from open check.

Reply:
Also want to make sure I do not need to defer a closeout here.